### ISSUE TITLE
Try-catch argument is not resolved inside catch block

### ISSUE THREAD

**Issue by eliasku** [TiVo/intellij-haxe#419]
Catch argument-variable is not resolved inside the catch block.
![image](https://cloud.githubusercontent.com/assets/3038174/13072124/907a269c-d4a8-11e5-864d-bf0168f0f740.png)

Repro:
```
class Test {
	public static function main() {
		try {}
		catch(err:String) {
			// `e` not resolved
			trace(err);
		}
	}
}
```

Version: 0.9.9, IDEA 15, OSX

**Comment by EBatTiVo** [comment]
Is it likely that err is unresolved because Error is a standard type?  We need to implicitly add the 'std' classpath to the list of imports checked when looking for classes to resolve to.  We have this same problem for String, Integer, Float, etc.

**Comment by eliasku** [comment]
I will check it right now.
No. It's reproduced even if we are imports `flash.errors.Error`.
`flash.errors.Error` exceptions are thrown by `flash` platform classes. So `Error` is not standard type.
Default exception object in Haxe is String. But even if exception type is `String`, the `import String;` statement doesn't help. :(

**Comment by EBatTiVo** [comment]
Merged PR #421: 23d8c2d
